packages/builder/src/pages/builder/app/[application]/data/datasource/[datasourceId]/_components/PromptQueryModal.svelte

```diff
@@ -0,0 +1,61 @@
+<script>
+  import { goto as gotoStore, params as paramsStore } from "@roxi/routify"
+  import { Modal, ModalContent, Body, Heading } from "@budibase/bbui"
+  import FontAwesomeIcon from "components/common/FontAwesomeIcon.svelte"
+
+  const handleOpen = (modal, params) => {
+    if (params["?promptQuery"] && modal?.show) {
+      modal.show()
+      history.replaceState({}, null, window.location.pathname)
+    }
+  }
+
+  let modal
+  $: params = $paramsStore
+  $: goto = $gotoStore
```

Comment:
Random question - what's the purpose of reassigning these stores to local variables?

Reply:
Good question, for whatever reason, if I'd just reloaded the page, I kept getting an error when I tried to use `get` on them in the callback 🤷 , no idea why